source/hassio/installation.markdown

```diff
@@ -17,7 +17,7 @@ The following will take you through the steps required to install Home Assistant
      - [Odroid-C2][odroid-c2], [Odroid-N2][odroid-n2], [Odroid-XU4][odroid-xu4]
      - [Intel NUC][intel-nuc]
 
-   - As a virtual appliance (x86_64/UEFI):
+   - As a virtual appliance (x86_64/UEFI image) for an virtual machine hypervisor:
```

Comment:
This is now double up. `virtual appliance` and `virtual machine hypervisor`. Please revert this change.